docs/user.md

```diff
@@ -242,3 +242,34 @@ metadata:
 
 Note that timezone required for `timestamp` (offset relative to UTC, see RFC
 3339 section 5.6)
+
+## Increase volume size
+
+PostgreSQL operator supports statefulset volume resize if you're using the
+operator on top of AWS. For that you need to apply manifest with a new size,
+and the operator will find out that a volume size needs to be changed:
+
+```
+apiVersion: "acid.zalan.do/v1"
+kind: postgresql
+
+metadata:
+  name: acid-test-cluster
+spec:
+  volume:
+    size: 5Gi # new volume size
+```
+
+You can only increase a volume size in this way, if a new requested size is
```

Comment:
You can only enlarge the volume with the process described above, shrinking is not supported and will emit a warning.